### PR TITLE
fix(chore): use double colons for pseudo-elements

### PR DESCRIPTION
**Pull request by grandeljay**
It's a minor nit-pick but as a FE Framework I think it's good to do things "the right way", even if one colon is probably also fine.

> As a rule, double colons (::) should be used instead of a single colon (:). This distinguishes pseudo-classes from pseudo-elements. However, since this distinction was not present in older versions of the W3C spec, most browsers support both syntaxes for the original pseudo-elements.

https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

### CODE REVIEW

**Review by lubber-de — CHANGES_REQUESTED** [review]
Please remove all changes inside the /dist folder from your PR, especially the minimized versions cannot really be reviewed and are generated upon a release anyway.

**Comment by grandeljay** [comment]
Wasn't sure about that since the dist folder isn't on the `.gitignore`. Should I add it? (Can so seperate PR for that)

**Comment by lubber-de** [comment]
> Wasn't sure about that since the `dist` folder isn't on the `.gitignore`. Should I add it? (Can do separate PR for that)

No, the dist folder will be part / is needed for an official release

**Comment by grandeljay** [comment]
Alrighty, hope this does the trick.

File: src/definitions/collections/menu.less [hunk 1]
```diff
@@ -384,10 +384,10 @@
 
   /* Show vertical dividers below last */
 
-  .ui.vertical.sidebar.menu > .item:first-child:before {
+  .ui.vertical.sidebar.menu > .item:first-child::before {
     display: block !important;
   }
-  .ui.vertical.sidebar.menu > .item::before {
+  .ui.vertical.sidebar.menu > .item::after {
     top: auto;
```
Comment (lubber-de) [inline src/definitions/collections/menu.less:391]: Why the change from ::before to ::after?

File: src/definitions/collections/menu.less [hunk 2]
```diff
@@ -518,15 +518,15 @@ Floated Menu / Item
 }
 
 /* Swapped Borders */
-.ui.menu .right.item::before,
-.ui.menu .right.menu > .item::before {
+.ui.menu .right.item::after,
```
Comment (lubber-de) [inline src/definitions/collections/menu.less:521]: Why the change from ::before to ::after?

File: src/definitions/collections/menu.less [hunk 3]
```diff
@@ -518,15 +518,15 @@ Floated Menu / Item
...
+.ui.menu .right.menu > .item::after {
   right: auto;
   left: 0;
 }
 
 /* Remove Outer Borders */
-.ui.menu .center.item:last-child::before,
-.ui.menu .center.menu > .item:last-child::before {
+.ui.menu .center.item:last-child::after,
+.ui.menu .center.menu > .item:last-child::after {
```
Comment (lubber-de) [inline src/definitions/collections/menu.less:529]: Why the change from ::before to ::after?

File: src/definitions/collections/menu.less [hunk 4]
```diff
@@ -1603,14 +1603,14 @@ Floated Menu / Item
...
-  .ui.compact.menu .item:last-child:before {
+  .ui.compact.menu .item:last-child::before {
     display: none;
   }
   & when (@variationMenuVertical) {
     .ui.compact.vertical.menu {
       width: auto !important;
     }
-    .ui.compact.vertical.menu .item:last-child::before {
+    .ui.compact.vertical.menu .item:last-child::after {
```
Comment (lubber-de) [inline src/definitions/collections/menu.less:1613]: Why the change from ::before to ::after?

**Comment by grandeljay** [comment]
Was my branch not up-to-date? I thought I pulled the upstream develop branch before creating my commits.

File: src/definitions/elements/list.less [hunk 5]
```diff
@@ -665,12 +665,12 @@ ol.ui.list ol li,
     .ui.horizontal.bulleted.list > .item:first-child {
       margin-left: 0;
     }
-    ul.ui.horizontal.bulleted.list li::before,
-    .ui.horizontal.bulleted.list > .item::before {
+    ul.ui.horizontal.bulleted.list li::after,
+    .ui.horizontal.bulleted.list > .item::after {
```
Comment (lubber-de) [inline src/definitions/elements/list.less:669]: Why the change from ::before to ::after?

File: src/definitions/elements/list.less [hunk 6]
```diff
@@ -665,12 +665,12 @@ ol.ui.list ol li,
...
       color: @horizontalBulletColor;
     }
-    ul.ui.horizontal.bulleted.list li:first-child::before,
-    .ui.horizontal.bulleted.list > .item:first-child::before {
+    ul.ui.horizontal.bulleted.list li:first-child::after,
+    .ui.horizontal.bulleted.list > .item:first-child::after {
```
Comment (lubber-de) [inline src/definitions/elements/list.less:673]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 7]
```diff
@@ -134,7 +134,7 @@
       Hover
 ---------------*/
 
-.ui.checkbox label:hover::before {
+.ui.checkbox label:hover::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:137]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 8]
```diff
@@ -147,7 +147,7 @@
       Down
 ---------------*/
 
-.ui.checkbox label:active::before {
+.ui.checkbox label:active::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:150]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 9]
```diff
@@ -419,26 +419,26 @@
   .ui.slider.checkbox label:hover {
     color: @sliderHoverLabelColor;
   }
-  .ui.slider.checkbox label:hover::before {
+  .ui.slider.checkbox label:hover::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:422]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 10]
```diff
@@ -610,7 +610,7 @@
   .ui.inverted.checkbox label:hover {
     color: @invertedHoveredTextColor !important;
   }
-  .ui.inverted.checkbox label:hover::before {
+  .ui.inverted.checkbox label:hover::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:613]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 11]
```diff
@@ -620,55 +620,55 @@
...
 
     /* Slider Line */
-    .ui.inverted.slider.checkbox label:before {
+    .ui.inverted.slider.checkbox label::before {
       background-color: @invertedUnselectedTextColor !important;
     }
 
     /* Slider Hover */
-    .ui.inverted.slider.checkbox label:hover::before {
+    .ui.inverted.slider.checkbox label:hover::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:628]: Why the change from ::before to ::after?

File: src/definitions/modules/checkbox.less [hunk 12]
```diff
@@ -620,55 +620,55 @@
...
   & when (@variationCheckboxToggle) {
     /* Toggle Switch */
-    .ui.inverted.toggle.checkbox label:before {
+    .ui.inverted.toggle.checkbox label::before {
       background-color: @invertedTextColor !important;
     }
 
     /* Toggle Hover */
-    .ui.inverted.toggle.checkbox label:hover::before {
+    .ui.inverted.toggle.checkbox label:hover::after {
```
Comment (lubber-de) [inline src/definitions/modules/checkbox.less:655]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 13]
```diff
@@ -452,13 +452,13 @@
     @c: @colors[@@color][color];
     @l: @colors[@@color][light];
 
-    .ui.indeterminate.@{color}.progress .bar::before,
+    .ui.indeterminate.@{color}.progress .bar::after,
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:455]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 14]
```diff
@@ -452,13 +452,13 @@
...
     .ui.@{color}.progress .bar,
     .ui.progress .@{color}.bar {
       background-color: @c;
     }
     & when (@variationProgressInverted) {
-      .ui.inverted.indeterminate.@{color}.progress .bar::before,
+      .ui.inverted.indeterminate.@{color}.progress .bar::after,
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:461]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 15]
```diff
@@ -504,7 +504,7 @@
     position: relative;
     right: 0;
   }
-  .ui.indeterminate.progress .bar::before {
+  .ui.indeterminate.progress .bar::after {
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:507]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 16]
```diff
@@ -515,31 +515,31 @@
...
       animation-duration: @indeterminatePulseDurationSlow;
     }
-    .ui.fast.indeterminate.progress .bar::before {
+    .ui.fast.indeterminate.progress .bar::after {
       animation-duration: @indeterminatePulseDurationFast;
     }
   }
   & when (@variationProgressSwinging) {
-    .ui.swinging.indeterminate.progress .bar::before {
+    .ui.swinging.indeterminate.progress .bar::after {
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:526]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 17]
```diff
@@ -515,31 +515,31 @@
...
   & when (@variationProgressSliding) {
-    .ui.sliding.indeterminate.progress .bar::before {
+    .ui.sliding.indeterminate.progress .bar::after {
       transform-origin: left;
       animation-name: progress-sliding;
     }
   }
   & when (@variationProgressFilling) {
-    .ui.filling.indeterminate.progress .bar::before {
+    .ui.filling.indeterminate.progress .bar::after {
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:538]: Why the change from ::before to ::after?

File: src/definitions/modules/progress.less [hunk 18]
```diff
@@ -515,31 +515,31 @@
...
       animation-name: progress-filling;
     }
   }
-  .ui.indeterminate.progress:not(.sliding):not(.filling):not(.swinging) .bar::before {
+  .ui.indeterminate.progress:not(.sliding):not(.filling):not(.swinging) .bar::after {
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:542]: Why the change from ::before to ::after?

**Review by lubber-de — CHANGES_REQUESTED** [review]
You replaced ::before by ::after at multiple places. That's why i thought the branch was not up to date. But unfortunatley it seems to be a copy/paste issue 😉

**Review by lubber-de — APPROVED** [review]
LGTM
